src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.SslProtocols.cs

```diff
@@ -84,7 +84,10 @@ public static IEnumerable<object[]> GetAsync_AllowedSSLVersion_Succeeds_MemberDa
 #pragma warning disable 0618
             if (PlatformDetection.IsWindows ||
                 PlatformDetection.IsOSX ||
-                (RuntimeInformation.IsOSPlatform(OSPlatform.Linux) && PlatformDetection.OpenSslVersion < new Version(1, 0, 2) && !PlatformDetection.IsDebian))
+                (RuntimeInformation.IsOSPlatform(OSPlatform.Linux) &&
+                 PlatformDetection.OpenSslVersion < new Version(1, 0, 2) &&
+                 !PlatformDetection.IsDebian &&
+                 !PlatformDetection.IsRedHatFamily6))
```

Comment:
Please add comments about why we are disabling the test, i.e. this information should be captured in a comment in the test method.  Otherwise, we'll never remember a year from now why we did it.

>Because certain applications perform incorrect version check of the OpenSSL version, the actual 
runtime version of OpenSSL is masked and the build-time version is reported instead. Consequently,
it is impossible to detect the currently running OpenSSL version using the SSLeay() function.

Reply:
I've added this in the latest commit.

Reply:
nit: openSsl -> OpenSSL
I'd remove this:
```
Some linux distributions intentionally misreport the current OpenSsl version for
compatibility reasons, and so the default behavior may vary from what we expect.
```
and put something like:
```
Some platforms don't support SSLv3 due to security issues (http://disablessl3.com/).
```